<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutTab</class>
 <widget class="QWidget" name="AboutTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>418</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="vboxMain">
   <item>
    <layout class="QHBoxLayout" name="hboxTitle">
     <item>
      <spacer name="hspcTitle1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblLogo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblTitle">
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspcTitle2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabCredits">
      <attribute name="title">
       <string>C&amp;redits</string>
      </attribute>
      <layout class="QVBoxLayout" name="vboxCreditsTab">
       <item>
        <widget class="QFrame" name="fraCredits">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="vboxFraCredits">
          <property name="margin">
           <number>8</number>
          </property>
          <item>
           <widget class="QScrollArea" name="scrlCredits">
            <property name="styleSheet">
             <string notr="true">QAbstractScrollArea {
	background-color: transparent;
}
QWidget#scrlCreditsContents {
	background-color: transparent;
} </string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrlCreditsContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>443</width>
               <height>313</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="vboxScrlCredits">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="lblCredits">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="textFormat">
                 <enum>Qt::RichText</enum>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="openExternalLinks">
                 <bool>true</bool>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLibraries">
      <attribute name="title">
       <string>&amp;Libraries</string>
      </attribute>
      <layout class="QVBoxLayout" name="vboxLibrariesTab">
       <item>
        <widget class="QFrame" name="fraLibraries">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="vboxFraLibraries">
          <property name="margin">
           <number>8</number>
          </property>
          <item>
           <widget class="QScrollArea" name="scrlLibraries">
            <property name="styleSheet">
             <string notr="true">QAbstractScrollArea {
	background-color: transparent;
}
QWidget#scrlLibrariesContents {
	background-color: transparent;
} </string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrlLibrariesContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>443</width>
               <height>313</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="vboxScrlLibraries">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="lblLibraries">
                <property name="textFormat">
                 <enum>Qt::RichText</enum>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="openExternalLinks">
                 <bool>true</bool>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSupport">
      <attribute name="title">
       <string>&amp;Support</string>
      </attribute>
      <layout class="QVBoxLayout" name="vboxSupportTab">
       <item>
        <widget class="QFrame" name="fraSupport">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="vboxFraSupport">
          <property name="margin">
           <number>8</number>
          </property>
          <item>
           <widget class="QScrollArea" name="scrlSupport">
            <property name="styleSheet">
             <string notr="true">QAbstractScrollArea {
	background-color: transparent;
}
QWidget#scrlSupportContents {
	background-color: transparent;
} </string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrlSupportContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>443</width>
               <height>313</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="vboxScrlSupport">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="lblSupport">
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="openExternalLinks">
                 <bool>true</bool>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>modified()</signal>
  <slot>reset()</slot>
  <slot>loadDefaults()</slot>
  <slot>save()</slot>
  <slot>checkBox_clicked()</slot>
 </slots>
</ui>
